canvas_modules/harness/cypress/integration/canvas/nodes.js

```diff
@@ -294,6 +294,30 @@ describe("Test from loaded file in legacy format", function() {
 	});
 });
 
+describe("Test new enableNodeLayout config parameter", function() {
+	beforeEach(() => {
+		cy.visit("/");
+		cy.openCanvasPalette("modelerPalette.json");
+	});
+
+	it("Test node 's height & width upon enableNodeLayout update", function() {
+		cy.clickToolbarPaletteOpen();
+		cy.clickCategory("Import");
+		cy.dragNodeToPosition("Database", 300, 200);
+
+		// Verify node height & width before updating enableNodeLayout
+		cy.verifyNodeTransform("Database", 8, 72);
+		cy.clickToolbarUndo();
+
+		// Override some of the enableNodeLayout config parameters
+		cy.setCanvasConfig({ "selectedNodeLayout": { defaultNodeWidth: 150, defaultNodeHeight: 150 } });
+		// Verify node height & width after updating enableNodeLayout
+		cy.dragNodeToPosition("Database");
+		cy.verifyNodeTransform("Database", 368, 234);
+	});
```

Comment:
These two lines above are not testing the height and width. The first is just dragging the node to a new position and the second is testing it is at that new position.  As mentioned above you'll need a function to test the height and width of the node.